### PR TITLE
Added More Tests

### PR DESCRIPTION
**Pull request by padmakarsrinivas**
Android Sauce Labs
iOS Sauce Labs

## Proposed changes

Added new sample tests for testing different Android and iOS Applications.

## Types of changes
Added new Sample Java Tests for Android and iOS Applications.

What types of changes does your code introduce to Appium?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

- [x] I have read the [CONTRIBUTING](https://github.com/appium/appium/blob/master/CONTRIBUTING.md) doc
- [x] I have signed the [CLA](https://cla.js.foundation/appium/appium)
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Review by saikrishna321 — CHANGES_REQUESTED** [review]
https://github.com/appium/appium/tree/master/sample-code/java/src

We already have tests for Android/iOS Native,Browser.

**Review by padmakarsrinivas — COMMENTED** [review]
Removed Native and Browser Test. Added Sauce Labs Tests.

File: sample-code/java/src/AndroidBrowserSaucelabsTest.java [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+public class AndroidBrowserSaucelabsTest extends BaseTest{
+    public static final String USERNAME = "YOUR_USERNAME";
+    public static final String ACCESS_KEY = "YOUR_ACESS_KEY";
+    public static final String URL = "https://"+USERNAME+":" + ACCESS_KEY + "@ondemand.saucelabs.com:443/wd/hub";
+    public static AndroidDriver<?> mobiledriver;
+
+    @BeforeTest
+    public void beforeTest( ) throws MalformedURLException {
+        DesiredCapabilities capabilities = new DesiredCapabilities();
+        capabilities.setCapability(MobileCapabilityType.APPIUM_VERSION, "1.7.2");
```
Comment (saikrishna321) [inline sample-code/java/src/AndroidBrowserSaucelabsTest.java]: We don't need appium version in caps.

File: sample-code/java/src/AndroidBrowserSaucelabsTest.java [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+        capabilities.setCapability(MobileCapabilityType.PLATFORM_VERSION, "4.4");
+        capabilities.setCapability(MobileCapabilityType.PLATFORM_NAME,"Android");
+        capabilities.setCapability(MobileCapabilityType.AUTOMATION_NAME,"Appium");
```
Comment (saikrishna321) [inline sample-code/java/src/AndroidBrowserSaucelabsTest.java]: AutomationName should be UIAutomator2

File: sample-code/java/src/IOSBrowserSaucelabsTest.java [hunk 3]
```diff
@@ -0,0 +1,48 @@
...
+    public static final String USERNAME = "YOUR_USERNAME";
+    public static final String ACCESS_KEY = "YOUR_ACESS_KEY";
+    public static final String URL = "https://"+USERNAME+":" + ACCESS_KEY + "@ondemand.saucelabs.com:443/wd/hub";
+    public static IOSDriver<?> mobiledriver;
+
+    @BeforeTest
+    public void beforeTest( ) throws MalformedURLException {
+        DesiredCapabilities capabilities = new DesiredCapabilities();
+
+        capabilities.setCapability(MobileCapabilityType.APPIUM_VERSION, "1.7.2");
```
Comment (saikrishna321) [inline sample-code/java/src/IOSBrowserSaucelabsTest.java]: Same here

**Review by saikrishna321 — CHANGES_REQUESTED** [review]

File: sample-code/java/src/IOSBrowserSaucelabsTest.java (hunk 3 above)
Comment (padmakarsrinivas) [reply sample-code/java/src/IOSBrowserSaucelabsTest.java]: Removed Appium Version and changed AutomationName to UIAutomator2

**Review by padmakarsrinivas — COMMENTED** [review]
Removed Appium Version and changed AutomationName to UIAutomator2

**Review by saikrishna321 — APPROVED** [review]

**Pull request merged**